registry/remote/credentials/internal/config/config_test.go

```diff
@@ -346,6 +346,47 @@ func TestConfig_GetCredential_emptyConfig(t *testing.T) {
 	}
 }
 
+func TestConfig_GetCredential_emptyFile(t *testing.T) {
```

Comment:
Just realized that the above test `TestConfig_GetCredential_emptyConfig` already does the same thing, but the referenced file name is wrong, oops. (Should be `empty.json` instead of `empty_config.json`)